### ISSUE TITLE
Is there a good reason that this package requires emacs 27.2?

### ISSUE THREAD

**Issue by loskool** [qhga/shanty-themes#5]
Gorgeous themes!

However, when I tried to install them through melpa, `package-install` complains that `Package 'emacs-27.2' is unavailable`. 

I'm using debian 11 and the emacs available through apt is still 27.1 

So, I manually installed `shanty-themes` into my `.emacs.d` directory and, when I load them, nothing seems to be broken. 

So is there a good reason to require 27.2? Doing so prevents, at the very least, debian 11 users from installing shanty themes from melpa.  

cheers

**Comment by qhga** [comment]
Hey @loskool,

thanks for your feedback! I am happy that you enjoy the themes so far. I actually decided to just use the version I used to develop the theme since time was quite limited back then. So no actual technical reason that I am aware of right now.

When I am back home, I'll chance the version to 27.1 or test the package for older emacs version as well and adjust the requirement accordingly.

Have a great day and thanks again for reporting!
